wini/scripts/sync-commit-hash.sh

```
@@ -0,0 +1,50 @@
+fi
+
+
+git remote add wini-template "$remote_url"
+git fetch wini-template
+
+remote_template_hashes="$(git log "wini-template/$branch" --pretty=format:"%H")"
+current_repo_hash="$(git log --pretty=format:"%H")"
+
+while IFS= read -r remote_hash; do
+    while IFS= read -r current_hash; do
+        if [[ $current_hash == $remote_hash ]]; then
```

Shellcheck (0.10.0)

[warning] Quote the right-hand side of == in [[ ]] to prevent glob matching.

(SC2053)